src/app/page.tsx

```diff
@@ -30,22 +29,9 @@ const getGoogleAnalyticsScript = () => {
   );
 };
 
-const { title, description, images, keywords } = metadataJSON.metadata.home;
-const canonical = `${HOST_URL}/`;
-const type = META_TYPE.WEBSITE;
-
-export const metadata: Metadata = {
-  metadataBase: new URL(HOST_URL),
```

Comment:
Can you check if omitting this attribute (`metadataBase`) triggers any base url related warning in the console. If so, find out a way solve that warning.